### ISSUE TITLE
As a user viewing a digitized work that's available in other editions, I want to know that other editions are available and have easy access to search across them.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/ppa-django#452]
## testing notes

on digitized work detail page on the public site
- [x] if work is part of cluster, it should have cluster icon and text
    - [x] search and browse link should take you to the search within the appropriate cluster
- [x] if work is _not_ in a cluster, it should not have the icon or text

## dev notes
detail view for a volume with other editions should have icon indicator and access to search across all editions

related to #445 and #451

**Comment by mnaydan** [comment]
@rlskoeser hm, I can't test the last one because I just noticed the interface is hiding texts that are NOT part of a cluster! I searched by title and author for specific texts I knew weren't part of a cluster and it failed to return the works. I confirmed this behavior by removing hvd.32044090278565 from association with "atestcluster2" on the backend, and the volume then disappeared from the public facing test site.
